includes/class-newspack-newsletters-editor.php

```diff
@@ -500,6 +500,9 @@ public static function maybe_exclude_sponsored_posts( $args, $request ) {
 					];
 				}
 			}
+		} else {
+			// If we're not removing sponsors by modifying the query, make sure to ignore sticky posts.
+			$args['ignore_sticky_posts'] = true;
```

Comment:
nit, but it seems to me that this method should be renamed and this clause should be always added...

Looks like this is needed in all cases, and it's odd that this lives just in a small `else` clause of something that is not completely related.

It could be called simply `rest_post_query_filter`, add this clause at the top and handle sponsors if needed.

Makes sense?

Reply:
Thanks Leo -- that makes sense! I went with this route since the changes we make to the query to remove sponsored posts also causes sticky posts not to stick, but it doesn't make for as clear of code. 

I went with the suggested more generic function in [3d2f3c1](https://github.com/Automattic/newspack-newsletters/pull/1814/commits/3d2f3c191b6a1bfdcb9bee8fc44094ec66b1dcd6). `maybe_filter_excerpt_length` also uses the same filter, but I left it out of the combined function it doesn't change the args, just filters the excerpt length, but let me know if it'd make sense to include, too!